zookeeper-server/src/test/java/org/apache/zookeeper/server/quorum/QuorumSSLTest.java

```diff
@@ -122,7 +124,21 @@
     private static final String PORT_UNIFICATION_DISABLED = "portUnification=false\n";
 
     private static final char[] PASSWORD = "testpass".toCharArray();
-    private static final String HOSTNAME = "localhost";
+    private static String HOSTNAME;
+    private static String ipAddress;
+
+    {
```

Comment:
is there a benefit using a static variable plus this initializer block? I would maybe prefer simply setting non-static `ipAddress` ad `hostname` in the `setup()` method we have already below. I think the later would be a bit nicer.

If you decide to stick with the static variables, then please use capital letters (e.g. IP_ADDRESS) to match the other static variables.